packages/discord-request/src/definitions.ts

```diff
@@ -0,0 +1,91 @@
+import type { Snowflake } from "discord-snowflake";
+
+/**
+ * @see {@link https://discord.com/developers/docs/resources/audit-log#audit-log-entry-object}
+ */
+export const AUDIT_LOG_LIMIT = 512;
+
+/**
+ * Helpers
+ */
+
+export type Prettify<T> = {
+	[K in keyof T]: T[K];
+} & {};
+
+/**
+ * Request Structures
+ */
+
+export enum RequestMethod {
+	Delete = "DELETE",
+	Get = "GET",
+	Patch = "PATCH",
+	Post = "POST",
+	Put = "PUT",
+}
+
+export interface Attachment {
+	id?: Snowflake;
+	name: string;
+	data: Blob;
+}
+
+export interface RequestOptions {
+	/**
+	 * Whether to use the authorization header.
+	 */
+	authorization?: boolean;
+
+	/**
+	 * The authorization prefix to use. Defaults to `Bot`.
+	 */
+	authorizationPrefix?: "Bot" | "Bearer";
```

Comment:
Do you need to support Bearer prefix?

Reply:
Probably not? This is a problem for future me

Reply:
Better to use `@defaultValue` than description for default values

Reply:
done